File: jijun0129/Algorithm/11.js
function solution(s) {
  stack = [s[0]];
  for (var i = 1; i < s.length; i++) {
    if (stack.length !== 0 && stack[stack.length - 1] === s[i]) {
      stack.pop();
    } else {
      stack.push(s[i]);
    }
  }
  if (stack.length === 0) {
    return 1;
  } else {
    return 0;
  }
}

File: seung365/programmers/11.py
def solution(s):
    answer = -1
    s_list = []
    index = 0
    s_list.append(s[0])
    for char in s[1:]:
        if s_list and s_list[-1] == char:
            s_list.pop()
            continue
        else:
            s_list.append(char)
            
    if s_list:
        answer = 0
    else:
        answer = 1

    return answer